docs/faas/faas-metrics.md

```diff
@@ -29,43 +37,435 @@ operations. By adding FaaS attributes to metric events it allows for finely tune
 
 ## Metric Instruments
 
-The following metric instruments MUST be used to describe FaaS operations. They MUST be of the specified
-type and units.
+The following metric instruments describe FaaS operations.
 
-### FaaS Invocations
+### Metric: `faas.invoke_duration`
 
-Below is a table of FaaS invocation metric instruments.
+This metric is [required][MetricRecommended].
```

Comment:
nit: it links to `Recommended` (and in other places in the doc)

Reply:
Good catch! Not a nit at all. Fixed.